### PR TITLE
Fix chat messages

### PR DESCRIPTION
**Pull request by KalmeMarq**
image

### CODE REVIEW

**Review by AwesomestCode — CHANGES_REQUESTED** [review]
Can we move the README changes to a separate PR?

**Comment by KalmeMarq** [comment]
wdym

File: lib/chat.js [hunk 1]
```diff
@@ -249,74 +243,86 @@ class ChatBox extends LitElement {
...
+        if (msg.extra) {
+          msg.extra.forEach(ex => {
+            readMsg(msglist, { ...styles, ...ex })
+          })
+        }
       }
+
+      readMsg(msglist, fullmessage)
+
+      console.log(fullmessage)
```
Comment (rom1504) [inline lib/chat.js:316]: this looks like forgotten debugging, can you remove it ?

**Comment by rom1504** [comment]
left a comment, otherwise lgtm

**Comment by KalmeMarq** [comment]
image

**Pull request merged**